{{cookiecutter.project_name}}/.mlops-setup-scripts/README.md

```diff
@@ -5,7 +5,9 @@ This directory contains setup scripts intended to automate CI/CD and ML resource
 for MLOps engineers.
 
 {% if cookiecutter.cicd_platform == "gitHub" -%}
-The scripts set up CI/CD with GitHub Actions. If using another CI/CD provider, you can
+The scripts set up CI/CD with GitHub Actions. 
+{% elif cookiecutter.cicd_platform == "gitHubEnterprise" -%}
```

Comment:
Lines 10 to 20 are previous under "gitHub". After the change, they will be only under "gitHubEnterprise" and missing from "gitHub".